README.md

```diff
@@ -11,6 +11,23 @@ An open language for communicating specifications on a printed circuit (mainly P
 ## Version
 Current version is 1.0. This should stated in every section directly below the "circuitdata" element in an element called "version".
 
+### Versioning
+CircuitData Language uses [semver](https://semver.org/). This means that we try our best to ensure that breaking changes mean a change in version number of the first component of the version.
+
+For example a change from `1.0` to `2.0` means that there have been breaking changes to the structure or allowed values in the language. If the version is changed by only the secondary component then it is a non breaking change. This means tooling compatible with the previous version should also be compatible with the next version. For example `1.1` to `1.2` is a non breaking change.
+
+### Releases of new versions
+
+The next version of the language is maintained on the master branch of this repository within the `schema/next` folder. Versions are created through the following process:
+1. Copy the files from `schema/next` to `schema/[version number]`.
+1. Bump the version number in this readme.
+1. Create a PR to make the change.
```

Comment:
Should there be a comment period prior to this PR being merged?

Reply:
not needed as long as all pull requests have been approved by the board.
All major changes goes to all board members - I am allowed by the board to take editorial and small substantial changes that don't affect functions or compatibility.

Reply:
Discussed this with Andreas and agreed that this makes sense for now.